3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,10 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+
+  const promArr = dice.map((x) => rollDie(x));
```

Comment:
Whenever you have a function in the format of `(arg,...) => fn(arg,...)` you can replace it with the function itself: `fn`. The arrow function would just be passing on the same argument list to called function without doing anything more.

```suggestion
  const promArr = dice.map(rollDie);
```